ci/check-style.sh

```diff
@@ -61,24 +50,41 @@ replace_original_if_changed() {
 
 # Apply cmake_format to all the CMake list files.
 #     https://github.com/cheshirekow/cmake_format
-find . \( "${ignore[@]}" \) -prune -o \
-  \( -name 'CMakeLists.txt' -o -name '*.cmake' \) \
-  -print0 |
-  xargs -0 cmake-format -i
+git ls-files | grep -P '(CMakeLists\.txt|\.cmake)$' | xargs cmake-format -i
```

Comment:
Precise translation of `-name 'CMakeLists.txt'` would be to match `(^|/)CMakeLists\.txt$` (i.e., not just a suffix).  Similarly for other full names below.

Reply:
Done.